### PR TITLE
Basic style Setup

### PR DESCRIPTION
**Pull request by nchen000**
### Overview of PR
- [x] added color palette
- [x] added 5 different font sizes (xs, sm, md, lg, xl)

### CODE REVIEW

File: src/index.scss [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+
+@import './theme.scss';
+
+html {
+  body {
+    margin: 0;
+    font-family: 'Roboto Condensed', sans-serif, monospace;
+    background: $light-gray;
+  }
 }
```
Comment (ethen001) [inline src/index.scss]: add new line ad the end of file

File: src/_theme.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+$dark-gray: #464646;
+$gray: #333333;
+$light-gray: #d2d2d2;
+$yellow: #f3cf45;
+
+$font-size-xs: 0.8em;
+$font-size-sm: 1em;
+$font-size-md: 1.2em;
+$font-size-lg: 1.5em;
+$font-size-xl: 1.8em;
```
Comment (ethen001) [inline src/_theme.scss]: add new line at end of file

File: src/_theme.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@mixin trans-background($color, $opacity: 0.9) {
+  background-color: $color;
+  background-color: rgba($color, $opacity);
+}
+
+$purple: #7d5cc6;
+$dark-gray: #464646;
+$gray: #333333;
+$light-gray: #d2d2d2;
+$yellow: #f3cf45;
```
Comment (JiejayLan) [inline src/_theme.scss]: I feel this is a strange combination of color. Purple, gray, yellow

Comment (nchen000) [reply src/_theme.scss]: This is the color scheme on CCNY website.

File: src/index.scss (hunk 1 above)
Comment (nchen000) [reply src/index.scss]: Since there is no warning showing up regarding EOL, there's already exists EOL on these files.

**Pull request merged**